src/os/FileStore.cc

```diff
@@ -2011,7 +2014,10 @@ int FileStore::queue_transactions(Sequencer *posr, list<Transaction*> &tls,
 
   //prepare and encode transactions data out of lock
   bufferlist tbl;
-  int data_align = _op_journal_transactions_prepare(tls, tbl);
+  int orig_len = -1;
+  if (journal && journal->is_writeable()) {
```

Comment:
Could assert(journal) because return 0 at 2011 if !journal.  I'm not clear on why is_writeable() is checked here when it wasn't before.